clients/youtube_producer/js/yt_auth/yt_authDirective.js

```diff
@@ -4,23 +4,10 @@ module.exports = function ($window, $http, $interval, $log) {
 		restrict: 'E',
 		link: function(scope, element, attr){
```

Comment:
I did not saw this really as a problem in the last pull request. But now I asked myself, why is this code on a directive? Is there some reason for that? Could it probably also run as a service, which is started in the run method of the angular module youtubeApp?